### PR TITLE
WW-427 WW_editInfoboxInVEbyPopup_fixedTtest

### PR DESCRIPTION
**Pull request by petrosinka**
fixed failing test
“portableinfoboxtests.PortableInfoboxTests.editInfoboxInVEbyPopup”

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java [hunk 1]
```diff
@@ -65,8 +65,10 @@
   private WebElement focusedHighlight;
   @FindBy(css = ".ve-init-mw-viewPageTarget-surface")
   private WebElement veSurface;
-  @FindBy(css = ".oo-ui-popupWidget-body .oo-ui-widget-enabled")
+  @FindBy(css = ".oo-ui-popupWidget-body .oo-ui-widget-enabled .oo-ui-iconElement")
   private WebElement infoboxPopup;
+  @FindBy(css=".pi-item.pi-item-spacing.pi-title")
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: Please update spacing - it's inconsistent with the rest of this file

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java [hunk 2]
```diff
@@ -505,12 +510,12 @@ public VisualEditorPageObject typeInParameterField(int i, String parameter) {
   public VisualEditorPageObject clickApplyChanges() {
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: I'm wondering about renaming this method to applyChanges() as it does more than just a click.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java [hunk 3]
```diff
@@ -559,4 +564,16 @@ public VisualEditorAddMediaDialog searchImage(String searchInput) {
     return mediaDialog;
   }
 
+  public VisualEditorPageObject  getTextArea() {
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: Strange spacing.
This method has a very misleading name, as it does not return a WebElement - it actually performs a click action on the text area. In this case, let's either rename it to clickTextArea() or return a webElement from it and perform a click action in the test body.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java [hunk 4]
```diff
@@ -559,4 +564,16 @@ public VisualEditorAddMediaDialog searchImage(String searchInput) {
...
+  }
+
+  public VisualEditorPageObject  getInfoboxNode() {
+    infoboxPopupCollapsed.click();
+    return this;
+  }
 }
+
+
+
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: Unneeded newlines. Please check the code style conventions for this pull request

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java [hunk 5]
```diff
@@ -559,4 +564,16 @@ public VisualEditorAddMediaDialog searchImage(String searchInput) {
     return mediaDialog;
   }
 
+  public VisualEditorPageObject  getTextArea() {
+    infoboxNode.click();
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: For this method to perform what we want it to we want it to be:
```
wait.forElementVisible(editArea);
editArea.click();
```

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java [hunk 6]
```diff
@@ -559,4 +564,16 @@ public VisualEditorAddMediaDialog searchImage(String searchInput) {
...
+    return this;
+  }
+
+  public VisualEditorPageObject  getInfoboxNode() {
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: Same as above method - it needs to be either clickInfobox() or clickInfoboxNode() or be split into getInfoboxNode() with a separate click

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java [hunk 7]
```diff
@@ -559,4 +564,16 @@ public VisualEditorAddMediaDialog searchImage(String searchInput) {
...
+    infoboxPopupCollapsed.click();
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: s/infoboxPopupCollapsed/infoboxNode/

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java (hunk 4 above)
Comment (petrosinka) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: done

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java (hunk 1 above)
Comment (petrosinka) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: done

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java (hunk 6 above)
Comment (petrosinka) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: done

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java [hunk 8]
```diff
@@ -95,6 +97,8 @@
   private List<WebElement> mediaNodes;
   @FindBy(css = ".media-gallery-wrapper.ve-ce-branchNode>div")
   private List<WebElement> galleryNodes;
+  @FindBy(css = ".ve-ce-surface.mw-body-content")
+  private WebElement infoboxNode;
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: This is now unused

Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java]: Addressed in f2f4c30

**Pull request merged**